Sequences/MRML/vtkMRMLNodeSequencer.cxx

```diff
@@ -139,6 +139,17 @@ void vtkMRMLNodeSequencer::NodeSequencer::AddDefaultDisplayNodes(vtkMRMLNode* no
   displayableNode->CreateDefaultDisplayNodes();
 }
 
+void vtkMRMLNodeSequencer::NodeSequencer::AddDefaultSequenceStorageNode(vtkMRMLNode* node)
+{
+  vtkMRMLStorableNode* storableNode = vtkMRMLStorableNode::SafeDownCast(node);
+  if (storableNode == NULL)
+  {
+    // not a storable node, there is nothing to do
+    return;
+  }
+  storableNode->CreateDefaultStorageNode();
```

Comment:
CreateDefaultStorageNode() just creates a storage node, but it is not added to the scene. I think we should use   AddDefaultStorageNode() here.